ui/app/AppLayouts/Chat/panels/InlineSelectorPanel.qml

```diff
@@ -58,15 +60,30 @@ Item {
                     color: Theme.palette.baseColor1
                 }
                 Item {
+                    id: tagsItemWrapper
                     Layout.fillWidth: true
                     Layout.fillHeight: true
+                    property real initWidth: 0.0
+                    Connections {
+                        target: Global
+                        function onOpenCreateChatView() {
+                            tagsItemWrapper.initWidth = width;
+                        }
+                    }
+                    onWidthChanged: {
+                        if ((initWidth !== width) && (scrollView.contentWidth > (parent.width))) {
+                            scrollView.contentX = scrollView.contentWidth - width + (width/3);
+                        } else {
+                            scrollView.contentX = 0
+                        }
+                    }
                     StatusScrollView {
                         id: scrollView
                         anchors.fill: parent
                         padding: 0
                         onContentWidthChanged: {
-                            if (scrollView.contentWidth > scrollView.width) {
-                                scrollView.contentX = scrollView.contentWidth - scrollView.width
+                            if (scrollView.contentWidth > (parent.width/2)) {
+                                scrollView.contentX = scrollView.contentWidth - parent.width + (warningLabel.visible ? (parent.width/2) : (parent.width/3));
```

Comment:
What is `parent.width/3`? Where does the dimension come from? Is it a warning label size? :eyes: